proc.go

```diff
@@ -116,6 +116,22 @@ func (p Proc) CmdLine() ([]string, error) {
 	return strings.Split(string(data[:len(data)-1]), string(byte(0))), nil
 }
 
+// Comm returns the command name of a process.
+func (p Proc) Comm() (string, error) {
+	f, err := os.Open(p.path("comm"))
```

Comment:
The indentation seems to be off by one level. Please run `go fmt` and change this commit.